packages/router/src/router.ts

```diff
@@ -519,30 +532,21 @@ export class Router {
   private ensureRootScope(): void {
     if (!this.rootScope) {
       const aureliaRootElement = this.container.get(Aurelia).root().$host;
-      this.rootScope = new Scope(this, aureliaRootElement as Element, null);
+      this.rootScope = new Scope(this, aureliaRootElement as Element, aureliaRootElement.$customElement.$context, null);
```

Comment:
```suggestion
      const root = this.container.get(Aurelia).root();
      this.rootScope = new Scope(this, root.$host as Element, root.$context, null);
```
(and of course remove the line above)